admin/services/data-access/pupils-not-taking-check.data.service.js

```diff
@@ -78,8 +78,8 @@ const pupilsNotTakingCheckDataService = {
            AND (
              -- No check has been generated for the pupil
                  p.currentCheckId IS NULL OR
-                 -- Or, a check was assigned but not received - cover new, collected and not received checks
-                 (p.currentCheckId IS NOT NULL AND p.checkComplete = 0 AND c.received = 0))
+                 -- Or, a check was assigned but is not complete - cover new, collected and not received checks
+                 (p.currentCheckId IS NOT NULL AND p.checkComplete = 0))
```

Comment:
I suspect this can be reduced to:

```sql
SELECT
	p.foreName,
	p.middleNames,
	p.lastName,
	p.dateOfBirth,
	p.urlSlug,
	p.group_id
FROM
	[mtc_admin].[pupil] p
WHERE
	p.attendanceId IS NULL
	AND p.checkComplete = 0
```

Reply:
it will need `p.school_id = @schoolId`